packages/router/src/grpc/offscreen-client.ts

```diff
@@ -48,7 +48,10 @@ const releaseOffscreen = async () => {
 const sendOffscreenMessage = async <T extends OffscreenMessage>(
   req: InternalRequest<T>,
 ): Promise<InternalResponse<T>> =>
-  chrome.runtime.sendMessage<InternalRequest<T>, InternalResponse<T>>(req);
+  chrome.runtime.sendMessage<InternalRequest<T>, InternalResponse<T>>(req).catch(() => ({
+    type: req.type,
+    data: null,
+  }));
```

Comment:
this doesn't suppress the serialized errors, which are thrown at https://github.com/penumbra-zone/web/blob/da3a476fc14c0e7c09c0b7bede07973841741029/packages/router/src/grpc/offscreen-client.ts#L80